src/plots/cartesian/axes.js

```diff
@@ -2167,7 +2177,11 @@ function numFormat(v, ax, fmtoverride, hover) {
         } else if(exponentFormat === 'B' && exponent === 9) {
             v += 'B';
         } else if(isSIFormat(exponentFormat)) {
-            v += SIPREFIXES[exponent / 3 + 5];
+            if(exponentFormat !== 'SI extended') {
+                v += SIPREFIXES[exponent / 3 + 5];
+            } else if(exponentFormat === 'SI extended') {
+                v += SIPREFIXES_EXTENDED[exponent / 3 + 10];
+            }
```

Comment:
You could switch to a ternary since there are only two options for this operation:
```suggestion
            v += exponentFormat === 'SI extended'
                ? SIPREFIXES_EXTENDED[exponent / 3 + 10]
                : SIPREFIXES[exponent / 3 + 5]
```